app/Http/Controllers/CategoryController.php

```diff
@@ -71,8 +71,9 @@ public function deleteCategory($id)
     public function showCategoryItem($id)
     {
         $category_items = Category::find($id)->item;
```

Comment:
you can remove this because it's overridden by the next line of code, or do you intend to use other variable?
$category_items = Cate gory::find($id)->item;

Reply:
Yes, I want to use the variable for the link of pagination.

Reply:
But I have something to modify, so I'll close this pull request. Thank you for checking.